ChatApp/Extensions/UIVIew+Extension.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  UIView+Extension.swift
+//  chatApp.test
+//
+//  Created by Ana Dzebniauri on 19.04.23.
+//
+
+import UIKit
+
+extension UIView {
+    func stretch(on parentView: UIView) {
+        NSLayoutConstraint.activate([
+            topAnchor.constraint(equalTo: parentView.topAnchor),
+            bottomAnchor.constraint(equalTo: parentView.bottomAnchor),
+            rightAnchor.constraint(equalTo: parentView.rightAnchor),
+            leftAnchor.constraint(equalTo: parentView.leftAnchor),
+        ])
+    }
+    
+    func stretchOnParent() {
+        guard let superview = superview else { return }
```

Comment:
you can also write like this: "guard let superview else { return }"

Reply:
same here Variable binding in a condition requires an initializer